athena-synapse/src/main/java/com/amazonaws/athena/connectors/synapse/SynapseJdbcConnectionFactory.java

```diff
@@ -60,25 +61,41 @@ public Connection getConnection(final CredentialsProvider credentialsProvider)
     {
         try {
             final String derivedJdbcString;
-            if (null != credentialsProvider) {
+            final Properties connectionProps = new Properties();
```

Comment:
Is there a reason why we still need the SynapseJdbcConnectionFactory? Can we not use the GenericConnectionFactory like we do in the other connectors now?